src/Pages/Expenses.cshtml.cs

```diff
@@ -49,14 +49,14 @@ public IActionResult OnPostNewEntry()
             }
             UserId = int.Parse(User.FindFirst(ClaimTypes.Name).Value);
             var entry = Entry.CreateLocalEntry(Title, Amount, Date, Recurring, Importance);
-            EntryDbUpdater.Add(entry, UserId, EntryType.Expense);
+            EntryDatabase.Create(entry, UserId, EntryType.Expense);
```

Comment:
Thanks for renaming it for me :D Im just a bit concerned that because I changed these parts a bit in my new branch there will be conflict, but nonetheless its just one line so everything should be fine :)

Reply:
Last change, I promise :D Sorry about these names, I just can't seem to make my mind on the best name for them, but I will just stick to what we have in here from here on out to stop the endless change